.github/workflows/report_news/main.py

```diff
@@ -58,6 +62,56 @@ def fetch_tech_griot() -> list:
     return articles
 
 
+def fetch_peef():
+    """
+    Fetching data from peef
+    """
+
+    url = f'{PEEF_SITEMAP_URL}'
+
+    with urlopen(url) as response:
+        body = response.read()
+
+    xml_raw = ET.fromstring(body)
+
+    articles = []
+    for url in xml_raw.findall('{http://www.sitemaps.org/schemas/sitemap/0.9}url')[-MAX_ARTICLES:]:
+        loc = url.find('{http://www.sitemaps.org/schemas/sitemap/0.9}loc')
+        lastmod = url.find('{http://www.sitemaps.org/schemas/sitemap/0.9}lastmod')
+
+        if loc is None or lastmod is None:
+            continue
+
+        link = re.sub(r'^http://', 'https://', urlquote(loc.text, safe=":/"))
+        pub_date = lastmod.text
```

Comment:
what's the purpose of creating this variable

Reply:
@Sanix-Darker, which variable?

Reply:
pub_date